app/src/main/java/com/infomaniak/mail/utils/Extensions.kt

```diff
@@ -70,6 +79,8 @@ fun View.setMargins(left: Int = 0, top: Int = 0, right: Int = 0, bottom: Int = 0
 
 fun String.isEmail(): Boolean = Patterns.EMAIL_ADDRESS.matcher(this).matches()
 
+fun String.isMe(): Boolean = AccountUtils.currentUser?.email == this
```

Comment:
Rename to `isMine` or something similar

Reply:
Has currently been copied to other branches that needed it as well, therefore I don't think it's the best moment to change its name. Maybe this could go into its own PR when we don't need to use it as much as currently